components/eventing-controller/controllers/subscription/beb/reconciler.go

```diff
@@ -852,6 +873,28 @@ func (r *Reconciler) checkStatusActive(subscription *eventingv1alpha1.Subscripti
 	return false, retry, err
 }
 
+// checkLastFailedDelivery checks if LastFailedDelivery exists and if happened after LastSuccessfulDelivery
+func (r *Reconciler) checkLastFailedDelivery(subscription *eventingv1alpha1.Subscription) bool {
+	if len(subscription.Status.EmsSubscriptionStatus.LastFailedDelivery) > 0 {
+		var lastFailedDeliveryTime, LastSuccessfulDeliveryTime time.Time
+		var err error
+		if lastFailedDeliveryTime, err = time.Parse(time.RFC3339, subscription.Status.EmsSubscriptionStatus.LastFailedDelivery); err != nil {
+			r.namedLogger().Errorw("parse LastFailedDelivery failed", "error", err)
```

Comment:
should we really hide this error here and assume everything is alright, by returning false?

Reply:
This error will be reflected in the subscription status too. It's an indicator that we have to update the "Webhook call status" condition to reflect the changes in BEB